livingatlas/pipelines/src/main/java/au/org/ala/pipelines/interpreters/SeedbankInterpreter.java

```diff
@@ -0,0 +1,289 @@
+      ExtensionInterpretation.extension(SEED_BANK_ROW_TYPE)
+          .to(SeedbankRecord::new)
+          .map(
+              SeedbankTerm.adjustedGerminationPercentage,
+              SeedbankInterpreter::setAdjustedGermination)
+          .map(SeedbankTerm.darkHours, SeedbankInterpreter::setDarkHours)
+          .map(SeedbankTerm.dayTemperatureInCelsius, SeedbankInterpreter::setDayTemp)
+          .map(
+              SeedbankTerm.nightTemperatureInCelsius,
+              SeedbankInterpreter::setNightTemperatureInCelsius)
+          .map(SeedbankTerm.lightHours, SeedbankInterpreter::setLightHours)
+          .map(SeedbankTerm.numberFull, SeedbankInterpreter::setNumberFull)
+          .map(SeedbankTerm.numberGerminated, SeedbankInterpreter::setNumberGerminated)
+          .map(SeedbankTerm.numberPlantsSampled, SeedbankInterpreter::setNumberPlantsSampled)
+          .map(SeedbankTerm.purityDebrisPercentage, SeedbankInterpreter::setPurityDebris)
+          .map(SeedbankTerm.sampleSize, SeedbankInterpreter::setSampleSize)
+          .map(SeedbankTerm.sampleWeightInGrams, SeedbankInterpreter::setSampleWeight)
+          .map(SeedbankTerm.testLengthInDays, SeedbankInterpreter::setTestLengthInDays)
+          .map(SeedbankTerm.thousandSeedWeight, SeedbankInterpreter::setThousandSeedWeight)
+          .map(SeedbankTerm.seedPerGram, SeedbankInterpreter::setSeedPerGram)
+          .map(SeedbankTerm.purityPercentage, SeedbankInterpreter::setPurityPercentage)
+          .map(SeedbankTerm.viabilityPercentage, SeedbankInterpreter::setViabilityPercentage)
+          .map(SeedbankTerm.relativeHumidityPercentage, SeedbankInterpreter::setRelativeHumidity)
+          .map(SeedbankTerm.storageTemperatureInCelsius, SeedbankInterpreter::setStorageTemp)
+          .map(SeedbankTerm.germinationRateInDays, SeedbankInterpreter::setGerminateRate)
+          .map(SeedbankTerm.numberEmpty, SeedbankInterpreter::setNumberEmpty)
+          .map(SeedbankTerm.numberTested, SeedbankInterpreter::setNumberTested)
+          .map(SeedbankTerm.dateInStorage, this::interpretDateInStorage)
+          .map(SeedbankTerm.dateCollected, this::interpretDateCollected)
+          .map(SeedbankTerm.testDateStarted, this::interpretTestDateStarted);
+
+  @Builder(buildMethodName = "create")
+  private SeedbankInterpreter(
+      List<DateComponentOrdering> orderings,
+      SerializableFunction<String, String> preprocessDateFn) {
+    this.temporalParser = TemporalParser.create(orderings);
+    this.preprocessDateFn = preprocessDateFn;
+  }
+
+  public void interpret(ExtendedRecord er, SeedbankRecord sr) {
+    Objects.requireNonNull(er);
+    Objects.requireNonNull(sr);
+    ExtensionInterpretation.Result<SeedbankRecord> result = HANDLER.convert(er);
+  }
+
+  public static void setSampleWeight(SeedbankRecord sr, String value) {
+    try {
+      sr.setSampleWeightInGrams(Double.parseDouble(value));
+    } catch (Exception e) {
+      // do nothing
```

Comment:
Should this and the other instances be adding an issue somewhere, since it's unexpected data?

Reply:
Yes, i think you are right. I'll create an issue for that